Add workspace authentication instructions (#71844)
Azure/azure-content: articles/machine-learning/tutorial-train-models-with-aml.md

print(ws.name, ws.location, ws.resource_group, sep='\t')
```

>[!NOTE]

> > It depends on your development environment (for example, your computer), you'll be asked to authenticate to your workspace by using a *device code* the first time you run the following code. Follow the on-screen instructions.

Reviewer: ```suggestion
> You may be asked to authenticate to your workspace the first time you run the following code. Follow the on-screen instructions.
```

Author: It become better😊. Let's do that.
